django_x509/base/admin.py

```diff
@@ -132,16 +131,6 @@ def get_urls(self):
             url(r'^x509/ca/(?P<pk>[^/]+).crl$', self.crl_view, name='crl')
         ] + super().get_urls()
 
-    def crl_view(self, request, pk):
-        authenticated = request.user.is_authenticated
-        authenticated = authenticated() if callable(authenticated) else authenticated
-        if app_settings.CRL_PROTECTED and not authenticated:
```

Comment:
@atb00ker See here, it has been checked that the user is authenticated or not.,but in `if` condition shouldn't any one of them be enough for this condition to be true and forbid the user fro accessing it. I checked with this default code by coping the url and fetching it after logging out the user. It still allows to download.

Reply:
https://github.com/openwisp/django-x509#django-x509-crl-protected
setting DJANGO_X509_CRL_PROTECTED to True will require authentication.